default/stringtables/en.txt

```diff
@@ -1147,6 +1147,19 @@ Gas Giants Habitable size
 RULE_HABITABLE_SIZE_DESC
 Value used to adjust for [[encyclopedia POPULATION_TITLE_SHORT_DESC]] changes on a planet of this size.
 
+RULE_THRESHOLD_HUMAN_PLAYER_WIN
+Max human players winners
+
+RULE_THRESHOLD_HUMAN_PLAYER_WIN_DESC
+Maximum number of human players that win together in multiplayer games if there are no other human player survivors.
+
+RULE_ONLY_ALLIANCE_WIN
+Only allied players win
+
+RULE_ONLY_ALLIANCE_WIN_DESC
+Allied players win together if the number of human players in their alliance is no large than the
+maximum human players winner and there are no other human player survivors.
```

Comment:
Do I should use text like following?
```
Allied players win together if the number of human players in their alliance is no large than the winning human survivors threshold and there are no other human player survivors.
```

Reply:
That looks fine to me.  

I also think it would be good to also make a slight adjustment to that underlying number of winners description to more clearly correspond (and just to slightly clarify in general), I would propose
```
+RULE_THRESHOLD_HUMAN_PLAYER_WIN_DESC
+Threshold number of human players that could win together in multiplayer games if there are no other human player survivors.
```

The two changes to the above text are to (i) add "could" before "win" (because if the rule requiring winners be in the same alliance is active, then there are not necessarily winners yet just because there are fewer than this number of human players left), and (ii)  change "Maximum" to "Threshold" (although "maximum" does seem technically accurate, it still just feels like a bit of an odd use here that takes extra thinking to figure out how it makes sense, and then using the word "threshold" also more closely mirrors the language used in the alliance rule.

Reply:
I'd prefer "maximum" actually... "threshold" to me means "minimum" when used like this.

Reply:
@Dilvish-fo Fixed but left "maximum" intact.